lib/rules/declaration-property-unit-allowed-list/README.md

```diff
@@ -83,3 +83,34 @@ a { animation-duration: 5s; }
 ```css
 a { line-height: 1; }
 ```
+
+## Optional secondary options
+
+### `ignore: ["inside-function"]`
+
+Ignore units that are inside a function.
+
+For example, given:
+
+```
+{
+  '/^border/': ['px'],
+  '/^background/': ['%'],
+}
```

Comment:
[suggestion]

- How about using double-quotes because users may assume JSON?
- How about describing full options for clarity?

```suggestion
[
  {
    "/^border/": ["px"],
    "/^background/": ["%"]
  },
  {
    "ignore": ["inside-function"]
  }
]
```

See also <https://github.com/stylelint/stylelint/blob/d4881680bbcca06d58aa6d4f48806e09a949c96e/lib/rules/declaration-property-unit-allowed-list/README.md#options>